backend/doc/cli.md

```diff
@@ -34,6 +34,22 @@ Useful to migrate old databases, to run before applying v3.0.0 migration scripts
 
     tracimcli db update-naming-conventions
 
+### Migrate Mysql/Mariadb database to utf8mb4 (added for tracim 3.7)
+
+:warning: This will modify the character set and collation of Tracim's database, including the already existing tables. We strongly suggest you to do a backup before applying this command.
+
+Useful to migrate old databases on mysql or mariadb in order to have full unicode support. This is mandatory starting with version 3.7 to support emoji reactions.
+
+For mysql 8.0.1+ (force collation to `utf8mb4_0900_ai_ci` as this is better than `utf8mb4_unicode_520_ci`):
+```bash
+tracimcli db migrate-mysql-charset -d --collation utf8mb4_0900_ai_ci
+```
+
+For mariadb 10.3+ (use collation `utf8mb4_unicode_520_ci` by default):
+```bash
+tracimcli db migrate-mysql-charset -d
+```
```

Comment:
Do you have an idea if on some existing installation, an error can be appear when starting migration ?

Reply:
Don't see much case where this can happened. you can have issue if you try to but an unknown charset or collation through this database, so it can failed:
- if you explicitly set an unavailable charset/collation on a database which has not this collation, the command for mysql 8.0.1 with `utf8mb4_0900_ai_ci` failed on mariadb because`utf8mb4_0900_ai_ci` don't exist on it.
- if in some new version of mariadb/mysql, one of them drop the support of collation `utf8mb4_unicode_520_ci` which is very unlikely.
- some other weird case of breaking chang upstream.